src/components/behandlerdialog/meldingtilbehandler/MeldingTilBehandlerSkjema.tsx

```diff
@@ -94,40 +65,29 @@ export const MeldingTilBehandlerSkjema = () => {
     });
   };
 
-  const validate = (
-    values: Partial<MeldingTilBehandlerSkjemaValues>
-  ): MeldingTilBehandlerSkjemaFeil => {
-    const meldingTilBehandlerSkjemaTekst = {
-      maxLength: MAX_LENGTH_BEHANDLER_MELDING,
-      value: values.meldingTekst || "",
-      missingRequiredMessage: texts.validation.missingMeldingTekst,
+  const submit = (values: MeldingTilBehandlerSkjemaValues) => {
+    const behandlerRefValue = () => {
+      if (
+        values.behandlerRef === "sokEtterBehandler" &&
+        selectedBehandler?.behandlerRef
+      ) {
+        return selectedBehandler?.behandlerRef;
+      } else {
+        return values.behandlerRef;
+      }
     };
```

Comment:
Siden ikke alle radiobuttons er valide valg i seg selv velges hvilken verdi vi skal bruke for å konstruere `MeldingTilBehandlerDTO` her